packages/ninjamsrv/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: jakob <grandchild@gmx.net>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=ninjam
+pkgname=ninjamsrv
+pkgver=0.080
+pkgrel=1
+pkgdesc='Ninjam online jamming server'
+arch=(x86_64 aarch64)
+url='https://cockos.com/ninjam/'
+license=(GPL2)
+depends=(gcc-libs)
+makedepends=()
+optdepends=()
+groups=(pro-audio)
+conflicts=(ninjam-server-git)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/cbix/$_name/archive/refs/tags/$pkgname/v$pkgver.tar.gz"
```

Comment:
How do you determine which commits you tag as releases?

Reply:
See my [comment here](https://github.com/justinfrankel/ninjam/issues/14#issuecomment-1166307136) :)

Reply:
Ok, thanks.